### ISSUE TITLE
🚧 Dev ➾ Migrate TZkT Docker Containers to ECAD Repo

### ISSUE THREAD

**Issue by hu3man** [ecadlabs/taqueria#1355]
### 🚥 Definition of Done

- [X] 🎯 Planning and Design
- [x] 🏎️ Dev Implementation (PR merged)
- [ ] ✍️ Release Notes Added
- [x] 🏆 Product Acceptance

### ⚖️ Acceptance Criteria

✨ Functional: 
- [ ] TzkT indexer and API docker images are owned/hosted by ECAD
- [ ] Docker images are integrated into the Taqueria CI 
 
 📐 Non-Functional: 
- [ ] Should be non-breaking for existing users
 

### 🔭 Scope

🙅‍♂️ Out of Scope: 
- Work on Taqueria/plugins
 
 

### 🧰 Work Description

Currently, some docker images related to the TzKT indexing functionality are owned/hosted on Ali's docker account. These should be moved over to an ECAD/Taqueria account and the images should be built and managed by the Taqueria CI/CD pipeline

### 🦄 Sizing

3

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by hu3man** [comment]
@GImbrailo, we are hoping to have you look into this before you leave

**Comment by GImbrailo** [comment]
Sure, I can take a look at this

**Comment by danielelisi** [comment]
@AlirezaHaghshenas where are the Dockerfiles hosted? Can you post links to your Docker Hub image page and your repo? 

If it's a private repo, please add Gino and me as contributors :smile:

**Comment by AlirezaHaghshenas** [comment]
I created [this issue](https://github.com/baking-bad/tzkt/issues/131) on TzKt's repo, requesting they migrate to dotnet 6 or 7, and asking if they will accept contribution.

Our change is in [my fork](https://github.com/AlirezaHaghshenas/tzkt/tree/taqueria) of their project (branch taqueria).
The images are release on: [Docker Hub](https://hub.docker.com/repository/docker/alirezahaghshenas/tzkt)

**Comment by AlirezaHaghshenas** [comment]
Update: they have been waiting for release of .net 7 (which was a few days ago) to migrate to it, as a problem in Json serialization in .net 6 was preventing them from migrating 5 to 6.
As soon as they migrate, we can test their image.

**Comment by danielelisi** [comment]
> Update: they have been waiting for release of .net 7 (which was a few days ago) to migrate to it, as a problem in Json serialization in .net 6 was preventing them from migrating 5 to 6. As soon as they migrate, we can test their image.

Thanks @AlirezaHaghshenas!

Should we still pursue migrating your fork to ECAD Labs repo, or can we wait for them on the `.net7` migration?

**Comment by GImbrailo** [comment]
@AlirezaHaghshenas has this issue become relevant again with the `.net7` migration?

FYI @hu3man @egarson

**Comment by AlirezaHaghshenas** [comment]
Depends on what the TzKt team decides to do with this PR: [Accept protocol Alpha, Override max attempts in config](https://github.com/baking-bad/tzkt/pull/139).

**Comment by AlirezaHaghshenas** [comment]
We are now using Baking Bad's official images after #1759
